patroni/postgresql.py

```diff
@@ -733,6 +757,25 @@ def get_pid(self):
             logger.warning("Garbage pid in postmaster.pid: {0!r}".format(pid))
             return 0
 
+    def get_pid_with_lost_data_dir(self):
+        logger.info("Trying to check if process running without directory "
+                    "with cached postmaster info: %s .", self._postmaster_info)
+        try:
+            process = psutil.Process(self._postmaster_info['pid'])
+            # check difference instead of values because of rounding issues
+            if abs(self._postmaster_info["start_time"] - process.create_time()) < 2:
+                return process.pid
+            else:
+                logger.warning("Process with pid %s was started at different time %s .",
+                               process.pid, process.create_time())
+        except psutil.NoSuchProcess:
+            logging.warning("Cannot find process %s .", self._postmaster_info['pid'])
```

Comment:
I think here should be an `info` message.
Also please be consistent and use "logger" everywhere (not logging).